samples/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/spotlightadvancedexample/SpotlightAdvancedExampleNode.kt

```diff
@@ -33,20 +37,16 @@ import com.bumble.appyx.navmodel.spotlightadvanced.operation.previous
 import com.bumble.appyx.navmodel.spotlightadvanced.operation.switchToCarousel
 import com.bumble.appyx.navmodel.spotlightadvanced.operation.switchToPager
 import com.bumble.appyx.navmodel.spotlightadvanced.transitionhandler.rememberSpotlightAdvancedSlider
-import com.bumble.appyx.sandbox.client.profilecard.ProfileCardNode
 import kotlinx.parcelize.Parcelize
 
 @Suppress("MaxLineLength")
 class SpotlightAdvancedExampleNode(
     buildContext: BuildContext,
     private val spotlightAdvanced: SpotlightAdvanced<NavTarget> = SpotlightAdvanced(
-        items = listOf(
-            NavTarget.Michaelle,
-            NavTarget.Lara,
-            NavTarget.Victoria,
-            NavTarget.Jessica,
-            NavTarget.Kane
-        ),
+        items = Profile.allProfiles
+            .shuffled()
+            .take(listOf(5, 7, 9, 11).random())
```

Comment:
This could easily break if the size of `allProfiles` becomes smaller. I guess it's fine for now, but maybe a safer solution would be better long term